### ISSUE TITLE
[req]: AnimeTime

### ISSUE THREAD

**Issue by garfield69** [Jackett/Jackett#14578]
### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Is there already a request for your tracker?

- [X] I have checked older issues, open and closed

### Type

Public

### Tracker details

```markdown
- Name: Anime Time
- URL: https://animetime.cc/
- Description: Anime Time is a PUBLIC torrent tracker for anime
- Minimum Seeding Time and Minimum Ratio (if Private): 
- Additional Comments:
```

**Comment by garfield69** [comment]
v0.21.532

**Comment by garfield69** [comment]
https://animetime.cc/ ERR_CONNECTION_REFUSED
http://animetime.cc/  no response data

**Comment by ilike2burnthing** [comment]
back up

**Comment by ilike2burnthing** [comment]
https://animetime.cc/
- HTTP ERROR 502
- so slow it times out in Jackett

**Comment by garfield69** [comment]
https://animetime.cc/ 502 Bad Gateway

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-1934504443

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://animetime.cc/ http timeout

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-1934504443

**Comment by garfield69** [comment]
online

**Comment by ilike2burnthing** [comment]
https://animetime.cc/ - Error 500, Internal Server Error

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
for just 4 minutes. I get timeout as do all uptime checkpoints ;-b

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-1934504443

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-1934504443

**Comment by ilike2burnthing** [comment]
back up

**Comment by ilike2burnthing** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-1974264218

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://animetime.cc/ HTTP ERROR 502

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-1974264218

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-1992128239

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-1992128239

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-1992128239

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-1934504443

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2086027980

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-1934504443

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://animetime.cc/
```
Notice of abscense
It has been around a year since we made public this site, we were not expecting so many traffic and we have been encountering with issues while trying to provide you the best service.

We have been monitoring the site since long and it has been going down for some minutes every few and then. Unfortunately we have not been able to identify and fix the root cause, so we have decided to go for a little while to start fresh in order to provide you the best experience.

See you soon!

TLDR; we are going for "vacations" for some time to improve the site and start fresh.
```

**Comment by ilike2burnthing** [comment]
We inadvertently DDoSing sites again? :P

**Comment by garfield69** [comment]
we already had a 2s delay, so hopefully not us.

**Comment by garfield69** [comment]
v0.22.372

**Comment by Noxillio** [comment]
Anime Time appears to be active once again. Will it be re-added?

**Comment by garfield69** [comment]
yes, will do.

**Comment by Noxillio** [comment]
What a quick response :) Thank you!

**Comment by garfield69** [comment]
v0.22.1316

**Comment by garfield69** [comment]
https://animetime.cc/ > ERR_CONNECTION_TIMED_OUT

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://animetime.cc/ > 502 Bad Gateway nginx

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2833536115

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://animetime.cc/ > 502 Bad Gateway nginx/1.22.1

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2936295948

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2936295948

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2936295948

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2936295948

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2936295948

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2936295948

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2936295948

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2936295948

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2936295948

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
- https://animetime.cc/ > Error 500, Internal Server Error Something went wrong on our site, please try again later
- 502 Bad Gateway

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-2936295948

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://animetime.cc/ > http 500 internal server error

**Comment by garfield69** [comment]
both domains seem to be online for half a day and i'm getting tired tracking these flaky sites.

**Comment by garfield69** [comment]
https://animetime.cc/ > 502 Bad Gateway    nginx/1.22.1

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14578#issuecomment-3199708374

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
Flaky as. Not reliably available.
Uptrends shows all checkpoints down with timeout.
Why are we keeping this indexer?

**Comment by ilike2burnthing** [comment]
When it comes back up, I'll see if it has anything the Nyaa trackers don't.

**Comment by garfield69** [comment]
online

**Comment by ilike2burnthing** [comment]
As far as I can tell, if it exists on Nyaa, it exists here. Can remove it or just stop monitoring it.

**Comment by garfield69** [comment]
v0.23.87
